setuptools_git_versioning.py

```diff
@@ -109,6 +112,8 @@ def parse_config(dist, _, value):  # type: (Distribution, Any, Any) -> None
     version_callback = value.get('version_callback', None)
     version_file = value.get('version_file', None)
     count_commits_from_version_file = value.get('count_commits_from_version_file', False)
+    global run_in_dir
+    run_in_dir = value.get('git_run_in_dir', None)
```

Comment:
`value` dict here is not all the `setup` function keyword arguments but a value of just `version_config` argument. So your example in the PR description will not work properly.

Reply:
Is there any way to get current package directory from `dist` argument? It'll be great not to pass any additional arguments to `setup` function at all.